src/main/java/me/arminb/sara/controller/models/AnswerRequest.java

```diff
@@ -0,0 +1,69 @@
+package me.arminb.sara.controller.models;
+
+import me.arminb.sara.entities.Answer;
+import me.arminb.sara.entities.Comment;
+
+import java.util.List;
+
+
+public class AnswerRequest {
+
+    private String answer;
+
+    private int rate;
+
+    private List<Comment> comments;
+
+    private String user;
```

Comment:
Generally userId should be taken from the request. For now it is okay cuz we havent implemented login yet. I would put a TODO everywhere I am doing this so I know where should be changed after the login functionality is implemented.